### PR TITLE
Use our repository for the CI

### PR DESCRIPTION
**Pull request by mudler**
Instead of polling mocaccino repositories, just install our toolset from our repository.

It also includes a commit that switches building nerdctl and k9s from source instead.

### CODE REVIEW

**Review by davidcassany — APPROVED** [review]
LGTM

File: packages/utils/jq/build.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  category: "distro"
+  version: ">=0"
+
+package_dir: /jq
+
+steps:
+- | 
+   PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+   mkdir -p /${PACKAGE_NAME}/usr/bin && \
+   curl -L https://github.com/stedolan/jq/releases/download/jq-${PACKAGE_VERSION}/jq-linux64 -o /${PACKAGE_NAME}/usr/bin/${PACKAGE_NAME} && \
```
Comment (Itxaka) [inline packages/utils/jq/build.yaml:12]: can we use `labels.github.owner` and `labels.github.repo` here? I found it pretty useful to test pre-release or patched releases from my own forks

**Review by Itxaka — APPROVED** [review]

File: packages/utils/jq/build.yaml (hunk 1 above)
Comment (mudler) [reply packages/utils/jq/build.yaml]: done!

**Comment by mudler** [comment]
Ok tests were green for the CI, I just had to resolve the conflict on the `manifest.yaml` file

**Pull request merged**